homeassistant/components/notify/webostv.py

```diff
@@ -21,12 +22,13 @@
 WEBOSTV_CONFIG_FILE = 'webostv.conf'
 HOME_ASSISTANT_ICON_PATH = os.path.join(os.path.dirname(__file__), '..',
                                         'frontend', 'www_static', 'icons',
-                                        'favicon-1024x1024.png')
+                                        'favicon-192x192.png')
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Optional(CONF_FILENAME, default=WEBOSTV_CONFIG_FILE): cv.string,
-    vol.Optional(CONF_ICON, default=HOME_ASSISTANT_ICON_PATH): cv.string
+    vol.Optional(CONF_ICON, default=HOME_ASSISTANT_ICON_PATH): cv.string,
+    vol.Optional(CONF_TIMEOUT, default=8): cv.positive_int
```

Comment:
We should not make timeout configurable. Instead, let's just always set it to 8.

Reply:
It is configurable on media_player, so I thought it would be nice to have in notify.

Reply:
It's not something that should be configurable.